cmd/jaeger-v2/config.yaml

```diff
@@ -0,0 +1,45 @@
+service:
+  extensions: [jaeger_storage, jaeger_query]
+  pipelines:
+    traces:
+      receivers: [otlp, jaeger, zipkin]
```

Comment:
Just one question, why we need `zipkin` receiver since it's been deprecated.

Reply:
Who says it was deprecated? There's plenty of zipkin instrumentation out there.

Reply:
Sorry I thought it was deprecated against OTEL instrumentation.